src/datachannel.cc

```diff
@@ -293,7 +298,13 @@ NAN_GETTER(DataChannel::GetBufferedAmount) {
 
   DataChannel* self = Nan::ObjectWrap::Unwrap<DataChannel>(info.Holder());
 
-  uint64_t buffered_amount = self->_jingleDataChannel->buffered_amount();
+  uint64_t buffered_amount;
+
+  if (self->_jingleDataChannel != nullptr) {
+    buffered_amount = self->_jingleDataChannel->buffered_amount();
+  } else {
+    buffered_amount = 0;
+  }
```

Comment:
WDYT about writing this as

```c++
uint64_t buffered_amount = self->_jingleDataChannel
  ? self->_jingleDataChannel->buffered_amount()
  : 0;
```

Also, in the future, this property should behave differently. See [here](http://w3c.github.io/webrtc-pc/#dom-rtcdatachannel-bufferedamount):

> the attribute does not reset to zero once the channel closes

That being said, the point of this PR is just to fix a segfault, not enforce spec-compliance, so we should punt this to a future PR.